src/components/select-option/select-option.tsx

```diff
@@ -97,7 +97,15 @@ export class SelectOption implements GxComponent, DisableableComponent {
 
   render() {
     return (
-      <Host aria-hidden="true" hidden={true}>
+      <Host
+        aria-hidden="true"
+        hidden={true}
+        selected={
+          this.element.parentElement.getAttribute("value") === this.value
+            ? "true"
+            : "false"
+        }
```

Comment:
Why are we setting one of the component's properties here?
It doesn't seem right. `Host` allows us to set standard attributes of the host element (like `style`, `class`, `role`, or `aria-*`), but setting properties of the own component doesn't seem a good practice. Instead, we should just set the property `this.selected = someValue`. If you want the property value to be reflected as an attribute, we should use the  `reflect` option of the `Prop()` decorator (more information about [reflect](https://stenciljs.com/docs/properties#reflect-properties-values-to-attributes)).

Reply:
By editing the `this.selected = true/false` will provoke another render execution, and then, another emittion of `gxDidLoad` event.
Setting as an attribute, the `gxSelect` will read the `gxSelectOption` with that attribute, and set it to the intern `option` of  intern `input select`.
Instead of setting the `selected` attribute manually, It's set automatically checking the `value` of `gxSelect`.

Reply:
Please bear in mind that `componentDidLoad` fires only after the first render, so no problem with that.

I think we could explore here setting `selected` from the parent component, that knows its value and "sees" the child components.

Reply:
Checkout changes about this in commit 7a92b8b